src/jquery.floatThead.js

```diff
@@ -95,6 +95,10 @@
         return result;
       };
     };
+    that.isFunction = function(obj) {
```

Comment:
thanks for the PR, but this isn't needed. it should be already handled above with the loop over `isThings` on line 71